nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java

```diff
@@ -114,6 +117,22 @@ public int getSerializationVersion() {
     return serializationAdapter.getSerializationVersion();
   }
 
+  /**
+   * Serializes the using {@link SerializationAdapter} version as {@code string} in
+   * <b>serialization_version.txt</b> file under root output directory for all serialized outputs.
+   *
+   * @param outputDirectory Path to root directory for all serialized outputs.
+   */
+  private void serializeVersion(@Nullable String outputDirectory) {
+    Path versionOutputPath = Paths.get(outputDirectory).resolve("serialization_version.txt");
+    try (Writer fileWriter =
+        Files.newBufferedWriter(versionOutputPath.toFile().toPath(), Charset.defaultCharset())) {
+      fileWriter.write(serializationAdapter.getSerializationVersion() + "");
```

Comment:
Why add an empty String?
```suggestion
      fileWriter.write(Integer.toString(serializationAdapter.getSerializationVersion()));
```